.kitchen.yml

```diff
@@ -14,8 +14,14 @@ platforms:
   - name: ubuntu-16.04
     run_list:
     - recipe[apt::default]
+  - name: centos-6.7
```

Comment:
You can remove 6.7 here. We just test on the latest image we have of each major release.